src/test/kotlin/kitchenpos/product/tobe/domain/ProductNameTest.kt

```diff
@@ -0,0 +1,42 @@
+package kitchenpos.product.tobe.domain
+
+import kitchenpos.product.tobe.infra.FakeProfanities
+import org.assertj.core.api.Assertions.assertThatIllegalArgumentException
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.CsvSource
+import org.junit.jupiter.params.provider.EmptySource
+
+class ProductNameTest {
+
+    @Test
+    @DisplayName("상품이름 생성 / 성공")
+    fun createProductName() {
+        val name = "양념치킨"
+        val productName = ProductName(profanities = FakeProfanities(), name = name)
+
+        assertEquals(name, productName.name)
+    }
+
+    @DisplayName("빈 이름 / 상품이름 생성 / 실패")
+    @EmptySource
+    @ParameterizedTest
+    fun createProductNameFail(name: String) {
+        assertThatIllegalArgumentException().isThrownBy {
+            ProductName(profanities = FakeProfanities(), name = name)
+        }
+    }
+
+    @DisplayName("욕설 / 상품이름 생성 / 실패")
+    @ParameterizedTest
+    @CsvSource(value = ["욕설상품", "비속어"])
```

Comment:
모델링 시 사용했던 문장을 활용하는것도 좋다고 생각해요!

>`Product`의 `name`은 필수값이고, `Profanities`를 통해 `Profanity`가 포함되어 있지 않은지 확인한다.

```suggestion
    @DisplayName("`Product`의 `Profanities`를 통해 `Profanity`가 포함된 것이 확인되면 상품 이름은 생성할 수 없다.`)
    @ParameterizedTest
    @CsvSource(value = ["욕설상품", "비속어"])
```

Reply:
모델링의 효용성을 더 높여주는 테스트 네이밍이네요!  새로 작성한 테스트에 전체적으로 반영해뒀어요👍

Reply:
요건 실수인데 백틱은 제외하고 작성해주셔도 됩니다! ㅎㅎ...
(수정은 안하셔도 괜찮아요!)